### DISCUSSION TITLE
Offline Karten

### DISCUSSION THREAD

**Discussion by 1maxi2** [devemux86/cruiser#72] [Q&A] [answered]
Gibt es in Cruiser als offline Karten nur " Mapsforge" ?
Mapsforge hat meiner Meinung ein etwas eigenartiges Design. 
Gibt es nicht die Möglichkeit die
 " Open topo Map " offline zu verwenden

**Comment by devemux86** [comment]
There are 3 offline map providers in the download map screen:
- Mapsforge
- [OpenAndroMaps](https://www.openandromaps.org/en)
- [Freizeitkarte](https://www.freizeitkarte-osm.de/android/en/index.html)

They can be used with the included or downloaded map themes to change the map appearance.

[OpenTopoMap](https://www.opentopomap.org) is an online bitmap map, so it cannot be used offline.

---

Es gibt 3 Offline-Kartenanbieter auf dem Bildschirm zum Herunterladen von Karten:
- Mapsforge
- [OpenAndroMaps](https://www.openandromaps.org)
- [Freizeitkarte](https://www.freizeitkarte-osm.de/android/de/index.html)

Sie können mit den mitgelieferten oder heruntergeladenen Kartenthemen verwendet werden, um das Aussehen der Karte zu verändern.

[OpenTopoMap](https://www.opentopomap.org) ist eine Online-Bitmap-Karte, kann also nicht offline verwendet werden.

**Comment by devemux86** [comment, marked as answer]
@1maxi2 

What exactly are you looking for, hill shading or contour lines?

---

Wonach suchen Sie genau, nach einer Hügelschattierung oder nach Konturlinien?

**Reply by 1maxi2** [reply]
Gerade die kleinen Strassen die interessant sind werden weiß auf weißen Hintergrund angezeigt und sind daher schwer zu erkennen.
Vielleicht mache ich einen Fehler. Bin kein Computer Spezialist 😏
<img src="https://user-images.githubusercontent.com/124886589/223777957-74bc3d17-b942-41d6-a279-ec334286f9a9.jpg" width="33%"/><img src="https://user-images.githubusercontent.com/124886589/223777965-b777aa4f-0116-4c6a-9891-5abdd147fb65.jpg" width="33%"/><img src="https://user-images.githubusercontent.com/124886589/223777971-1ab3c5f6-87de-409a-84c3-7b6f59287458.jpg" width="33%"/>

**Reply by devemux86** [reply]
_(I edited your post to display the images at a smaller size)_

---

_(Ich habe Ihren Beitrag so bearbeitet, dass die Bilder kleiner angezeigt werden)_

**Reply by devemux86** [reply]
You can change the map theme in map menu, for example the "Biker" theme:

---

Sie können das Kartenthema im Kartenmenü ändern, zum Beispiel das Thema "Biker":

<img src="https://user-images.githubusercontent.com/3484020/223781227-36bd0143-e98c-4a02-9a5b-22c2c6b124ad.jpg" width="50%"/><img src="https://user-images.githubusercontent.com/3484020/223781206-c50bb88b-52b5-4adf-965b-1fde868c1319.jpg" width="50%"/>

**Reply by 1maxi2** [reply]
Vielen Dank für die Hilfe ! Jetzt hat es geklappt. Sorry für meine blöden Fragen - bin noch an die Bedienung von kurviger gewöhnt .

**Comment by devemux86** [comment]
In the next Cruiser update, the "Biker" map theme will be the default for new installations.

So users will be able to find and use it more easily.

---

Im nächsten Cruiser-Update wird das Kartenthema "Biker" zum Standard für neue Installationen.

So können Benutzer es leichter finden und verwenden.

**Reply by 1maxi2** [reply]
👍
